src/shared/api/services/ActivityItemDto.ts

```diff
@@ -215,6 +216,15 @@ type MultiSelectionAnswerSettings = {
   options: OptionsDto;
 };
 
+type ParagraphTextConfiguration = ButtonsConfiguration &
+  TimerConfiguration & {
+    maxResponseLength: number;
+    correctAnswerRequired: boolean;
+    correctAnswer: string;
+    numericalResponseRequired: boolean;
+    responseDataIdentifier: boolean;
```

Comment:
Github ate my comment

These options are not available to `ParagraphItem` items

| Text Item | Paragraph Item |
| - | - |
| ![CleanShot 2024-07-30 at 17 16 46](https://github.com/user-attachments/assets/5eb3176e-be10-47eb-b1d9-4bbdfd810f1b) |  ![CleanShot 2024-07-30 at 17 16 40](https://github.com/user-attachments/assets/a9e4708f-700a-48fa-8ab6-865180c504ed) |